lib/crontab/cron_expression/parser.ex

```diff
@@ -72,32 +72,31 @@ defmodule Crontab.CronExpression.Parser do
       iex> Crontab.CronExpression.Parser.parse "* * * * *"
       {:ok,
         %Crontab.CronExpression{day: [:*], hour: [:*], minute: [:*],
-        month: [:*], weekday: [:*], year: [:*]}}
+        month: [:*], weekday: [:*], year: [:*], on_ambiguity: [:later]}}
 
       iex> Crontab.CronExpression.Parser.parse "* * * * *", true
       {:ok,
         %Crontab.CronExpression{extended: true, day: [:*], hour: [:*], minute: [:*],
-        month: [:*], weekday: [:*], year: [:*], second: [:*]}}
+        month: [:*], weekday: [:*], year: [:*], second: [:*], on_ambiguity: [:later]}}
 
       iex> Crontab.CronExpression.Parser.parse "fooo"
       {:error, "Can't parse fooo as minute."}
 
   """
-  @spec parse(binary, boolean) :: result
-  def parse(cron_expression, extended \\ false)
+  @spec parse(binary, boolean, [CronExpression.ambiguity_opt()]) :: result
+  def parse(cron_expression, extended \\ false, ambiguity_opts \\ [:later])
 
-  def parse("@" <> identifier, _) do
+  def parse("@" <> identifier, _, _) do
```

Comment:
I believe we also need to pass the ambiguity options to specials. For example most of them run at midnight. There's no guarantee that a TZ change does not happen over midnight.